shotgun_api3/shotgun.py

```diff
@@ -3346,10 +3358,15 @@ def _make_call(self, verb, path, body, headers):
                 if attempt == max_rpc_attempts:
                     raise
             except Exception:
-                #TODO: LOG ?
```

Comment:
It's not your code, but I'm really worried about this `except` clause. Any errors raises by the code, even programming errors on our part, will retry the request. This seems dangerous to me.

@thebeeland , should we log a tech debt ticket about this and figure out how to properly fix it? We'd have to take into account all the different http and socket error types that can be encountered here.

Reply:
We should, and I will. Thanks for the heads up.